cfg/std.cfg

```diff
@@ -7206,6 +7206,17 @@ initializer list (7) string& replace (const_iterator i1, const_iterator i2, init
       <not-uninit/>
     </arg>
   </function>
+  <!-- template< class OutputIt, class Size, class T > void std::fill_n( OutputIt first, Size count, const T& value ); // until C++11 -->
+  <!-- template< class OutputIt, class Size, class T > OutputIt std::fill_n( OutputIt first, Size count, const T& value ); // since C++11, until C++20 -->
+  <!-- template< class OutputIt, class Size, class T > constexpr OutputIt std::fill_n( OutputIt first, Size count, const T& value ); // since C++20 -->
+  <!-- template< class ExecutionPolicy, class ForwardIt, class Size, class T > ForwardIt std::fill_n( ExecutionPolicy&& policy, ForwardIt first, Size count, const T& value ); // since C++17 -->
+  <function name="std::fill_n">
+    <noreturn>false</noreturn>
+    <arg nr="1"/>
```

Comment:
`direction="int"`?

Reply:
Hmm, i am not sure. The output iterator points to content that is changed. It is not the same as a pointer but isn't it similar? Maybe it would even make sense to mark it with "out"? As far as I know iterators are not yet handled regarding the direction of the argument. I would not specify any direction attribute until someone can say for sure how it is correct.